extensions/signature.md

```diff
@@ -0,0 +1,108 @@
+# Signature Extension
+
+This extension defines a standard mechanism to carry signatures allowing
```

Comment:
In a typical scenario, where the events generated could be relayed by middleware to the eventual consumers, would it be appropriate to ask about multiple signers?

Reply:
this seems similar to comments made by @erikerikson.

I'm open to suggestions as to the best way to represent this with a flat attribute structure...